packages/alias/src/index.js

```diff
@@ -55,7 +44,6 @@ const getEntries = ({ entries }) => {
 };
 
 export default function alias(options = {}) {
-  const resolve = Array.isArray(options.resolve) ? options.resolve : ['.js'];
```

Comment:
👍 for a whole change, always have found it quirky for this plugin to handle this

Reply:
Great, then can i resolve this conversation?